permission_validators/main/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "iroha_main_validator"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+# A smartcontract should be linked dynamically so that it may link to functions exported
+# from the host environment. Also, host environment executes the smartcontract by
+# calling the function which smartcontract exports (entry point of execution)
+crate-type = ['cdylib']
+
+[dependencies]
+iroha_wasm = { path = "../../wasm", features = ["debug"]}
```

Comment:
have a look [here](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table) on how you can inherit dependencies from the workspace as well